example_project/assets/ts_config_example/vite.config.mts

```diff
@@ -0,0 +1,82 @@
+/**
+ * This file is provided as an example of how to configure Vite for a 
+ * Phoenix LiveView project, using typescript. If you want to use this, you can 
+ * replace assets/vite.config.js with this file.
+ * 
+ * Some Vite plugins don't have a very good typescript support so it's hard
+ * to get rid of the the cast to PluginOption[] in the plugins array.
+ */
+
+import path from "path"
+import { defineConfig, PluginOption } from "vite"
+
+import vue from "@vitejs/plugin-vue"
+import autoprefixer from "autoprefixer"
+import liveVuePlugin from "live_vue/vitePlugin"
+import tailwindcss from "tailwindcss"
+import { fileURLToPath } from "url"
+import vuetify from "vite-plugin-vuetify"
+import tailwindConfig from "./tailwind.config.ts"
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+// https://vitejs.dev/config/
+export default defineConfig(({ command }) => {
+  const isDev = command !== "build"
+
+  return {
+    base: isDev ? undefined : "/assets",
+    publicDir: "static",
+    // when using typescript in your vite config, you must delete the 
+    // postcss.config.js file, and rely on the vite css option instead
+    // to load the autoprefixer and tailwindcss plugins
+    css: {
+      postcss: { plugins: [tailwindcss(tailwindConfig), autoprefixer()] },
+    },
+    plugins: [
+      vue(),
+      // if you change ./js/server.js, to typescript, you need
+      // to pass this entrypoint option
+      liveVuePlugin({ entrypoint: "./js/server.ts" }),
```

Comment:
I think this is one last TS leftover?

Reply:
I left this bit, because if someone wants to use the vite ts config and server.ts (which works great), then this is a good hint. 

this is not used by the example project which relies on the assets/vite.config.js file I restored (and which is the one currently in main, pointing to server.js)

Reply:
You're right, I was checking the code from mobile and I thought it was in the main file. Anyway you left a comment so it should be just fine 👍